internal/state/db/statepsql/members.go

```diff
@@ -333,5 +333,31 @@ func (db *db) GetUserInGuildHasRole(ctx context.Context, guildID int64, roleID i
 		return false, xerrors.Errorf("exec select: %w", err)
 	}
 
+	return exists, nil
+}
+
+func (db *db) GetUserInGuildsHasRoles(ctx context.Context, guildIDs []int64, roleIDs []int64, userID int64) (bool, error) {
```

Comment:
can change func name to `ExistUserInGuildsHasRoles`

Reply:
Will change